tests/integration/test_sql_mail_template.py

```
@@ -0,0 +1,37 @@
+async def test_get_mail_template_success(postgres_session):
+    from src.infra.cache.mail_template_cache import MailTemplateCache
+    from src.infra.db.sql.orm.mail_template_orm import MailTemplate
+
+    # 插入測試資料
+    template = MailTemplate(
+        id="auth_template",
+        content="<h1>Hello {{ name }}</h1>",
+        name="auth",
+    )
+    postgres_session.add(template)
+    await postgres_session.flush()  # 寫入但不 commit，測試結束後自動 rollback
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)